src/components/import/badges/DevPreviewBadge.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { Label } from '@patternfly/react-core';
+import './Badge.scss';
+import { useTranslation } from 'react-i18next';
+
+const DevPreviewBadge: React.FC = () => {
+  const { t } = useTranslation();
+  return <Label className="ocs-preview-badge">{t('console-shared~Dev preview')}</Label>;
```

Comment:
`ocs` prefix here again.  I think we will probably have to externalize this string as our own, and not use `console-shared`.

Reply:
Will fix this in [GITOPS-2291](https://issues.redhat.com/browse/GITOPS-2291).